insecure/wintermute/attackOrchestrator.go

```diff
@@ -34,7 +56,244 @@ func (o *Orchestrator) WithAttackNetwork(network insecure.AttackNetwork) {
 // In Corruptible Conduit Framework for BFT testing, corrupted nodes relay their outgoing events to
 // the attacker instead of dispatching them to the network.
 func (o *Orchestrator) HandleEventFromCorruptedNode(event *insecure.Event) error {
+	o.Lock()
+	defer o.Unlock()
+
+	switch protocolEvent := event.FlowProtocolEvent.(type) {
+
+	case *flow.ExecutionReceipt:
+		if err := o.handleExecutionReceiptEvent(event); err != nil {
+			return fmt.Errorf("could not handle execution receipt event: %w", err)
+		}
+	case *messages.ChunkDataRequest:
+		if err := o.handleChunkDataPackRequestEvent(event); err != nil {
+			return fmt.Errorf("could not handle chunk data pack request event: %w", err)
+		}
+	case *messages.ChunkDataResponse:
```

Comment:
can we add this comment to elaborate this case:
`// orchestrator received chunk data response from EN when EN wants to respond to a chunk data request from VN`

Reply:
https://github.com/onflow/flow-go/pull/2291/commits/e752e4fd7d551717520bec62d1d135e957815633